recipe/build.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/bash
 
+ if [ $(uname) == Darwin ]; then
```

Comment:
I put it there temporarily while we wait for https://github.com/conda-forge/conda-smithy/pull/114.

But you are right. It must be removed ASAP. Changes are I will forget it there!